deploy/template.yaml

```diff
@@ -378,6 +378,7 @@ Resources:
               IDENTITY_KIWI_URL: "/perfTest/identity/kiwi/url"
               IDENTITY_KIWI_STUB_URL: "/perfTest/identity/kiwi/stubUrl"
               IDENTITY_KIWI_TARGET: "/perfTest/identity/kiwi/target"
+              IDENTITY_KIWI_STUB_SQS: "/perfTest/identity/kiwi/stubTxMAConsumer"
```

Comment:
- [ ] @abhinaysrivastavaperf Add `/perfTest/identity/kiwi/stubTxMAConsumer` to the paramater store before the PR is merged

Reply:
Yeah this has been added already. Thank you.